### PR TITLE
CI-beginner user example

### PR DESCRIPTION
**Pull request by jorgepz**
I am proposing to add a very simple/minimal config example for CI-beginner users.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -25,6 +25,34 @@ Package Name  | Version | Installation directory
...
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: joergbrech/moxunit-action@v1.1
```
Comment (joergbrech) [inline README.md:48]: Could you just fix the major version here?

```
- uses: joergbrech/moxunit-action@v1
```

I think its about time for a 1.2 release and then the README would be outdated.

**Review by joergbrech — CHANGES_REQUESTED** [review]
Awesome, good idea. Thanks for the contribution. 💐  I just have one very minor comment. If you fix this I will merge.

File: README.md (hunk 1 above)
Comment (jorgepz) [reply README.md:48]: ok, thanks for the fix! I am doings test in this repo https://github.com/jorgepz/Octave-Matlab-CI-testbed and it seems that both v1 and v1.1 work fine (at least for the minimal test I am running).
I'll fix my file.

**Review by joergbrech — APPROVED** [review]

**Comment by joergbrech** [comment]
@jorgepz thanks!

**Comment by jorgepz** [comment]
You are welcome. Congratulations for the idea of the repo. Very useful!

**Pull request merged**